doc/tutorials/acrn-secure-boot-with-grub.rst

```diff
@@ -0,0 +1,256 @@
+.. _how-to-enable-acrn-secure-boot-with-grub:
+
+Enable ACRN Secure Boot with GRUB
+#################################
+
+This document shows how to enable ACRN secure boot with GRUB including:
+
+-  ACRN Secure Boot Sequence
+-  Generate GPG Key
+-  Setup Standalone GRUB EFI Binary
+-  Enable UEFI Secure Boot
+
+**Validation Environment:**
+
+- Hardware Platform: WHL-IPC-I7, Supported hardware described in
+  :ref:`hardware`.
+- ACRN Scenario: Industry
+- Service VM: Yocto
```

Comment:
This mentions Yocto, although I believe it mentionned Ubuntu previously.

The script below still seems to be for Ubuntu, e.g. `TARGET_EFI='/boot/efi/EFI/ubuntu/grubx64.efi'`

Reply:
@gvancuts , tested on both yoctor and ubuntu, have update this mismatch, thanks.